files/default/sql/ddl/2.5.0__initial_tables.sql

```diff
@@ -2064,3 +2064,25 @@ CREATE TABLE IF NOT EXISTS `transformation_function` (
                                                          CONSTRAINT `feature_store_fn_fk` FOREIGN KEY (`feature_store_id`) REFERENCES `feature_store` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION,
                                                          CONSTRAINT `creator_fn_fk` FOREIGN KEY (`creator`) REFERENCES `users` (`uid`) ON DELETE NO ACTION ON UPDATE NO ACTION
 ) ENGINE = ndbcluster DEFAULT CHARSET = latin1 COLLATE = latin1_general_cs;
+
+CREATE TABLE IF NOT EXISTS `hopsworks`.`training_dataset_filter` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
+    `training_dataset_id` INT(11) NULL,
+    `type` VARCHAR(63) NULL,
+    `path` VARCHAR(63) NULL,
+    PRIMARY KEY (`id`),
+    CONSTRAINT `tdf_training_dataset_fk` FOREIGN KEY (`training_dataset_id`) REFERENCES `training_dataset` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+CREATE TABLE IF NOT EXISTS `hopsworks`.`training_dataset_filter_condition` (
+    `id` INT(11) NOT NULL AUTO_INCREMENT,
+    `training_dataset_filter_id` INT(11) NULL,
+    `feature_group_id` INT(11) NULL,
+    `feature` VARCHAR(63) NULL,
```

Comment:
would you mind calling this `feature_name` just to make it more intuitive.

Reply:
changed